packages/react-components/react-virtualizer/src/components/Virtualizer/useVirtualizer.ts

```diff
@@ -397,7 +400,15 @@ export function useVirtualizer_unstable(props: VirtualizerProps): VirtualizerSta
     populateSizeArrays();
   }
 
+  // Ensure we recalc if virtualizer length changes
+  const maxCompare = Math.min(virtualizerLength, numItems);
+  if (childArray.current.length !== maxCompare && virtualizerStartIndex + childArray.current.length < numItems) {
+    updateChildRows(virtualizerStartIndex);
+  }
```

Comment:
not sure how this change relates to the scrollview, is this a separate bugfix?

Reply:
This is a bugfix that affects resized windows & virtualizer length, we weren't seeing it before we added the resize observer as it was static virtualizer length.